src/components/common/cards/CaseStudyCard/CaseStudyCard.module.scss

```
@@ -0,0 +1,199 @@
+@import "src/resources/styles/colors";
+@import "src/resources/styles/sizes";
+@import "src/resources/styles/breakpoints";
+
+.caseStudyCard {
+  width: 260px;
+  height: 240px;
+
+  padding-bottom: 0;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  position: relative;
+  border-radius: 10px;
+
+  box-shadow: 0 5px 16px $color-card-shadow-soft;
+
+  @include desktop {
+    width: 1000px;
+    height: 296px;
+
+    padding: 0;
+    flex-direction: row;
+
+    border-radius: 10px 50px 10px 10px;
+    box-shadow: 0 10px 30px $color-card-shadow-soft;
+  }
+
+  .card-decorator {
+    width: 10px;
+
+    background: $gradient-card-decorator;
+    border-radius: 10px 0 0 10px;
+
+    @include mobile {
+      display: none;
+    }
+
+  }
+
+  .icon-container {
+    width: 72px;
+    height: 72px;
+
+    position: absolute;
+    top: -36px;
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
+
+    display: flex;
+    align-items: center;
+
+    background: $gradient-card;
+    border-radius: 50%;
+    border: 4px solid $color-surface-light;
+
+    box-shadow: 0 5px 16px $color-shadow;
+
+    @include desktop {
+      width: 128px;
+      height: 128px;
+
+      top: 32px;
+      left: -64px;
```

Can't we use a relative unit?

```suggestion
      left: -50vw;
```

---

Since we have a fixed width, it's more a matter of style, isn't it? I don't mind changing it if you prefer this way.

---

@yagotome this suggestion broke the icon component! We're using absolute positioning. The `vw` unit is not recognizing its parent. I rolled back to what we had before.